TROUBLESHOOTING.md

```diff
@@ -1,60 +1,63 @@
 # Troubleshooting
 
-* [Tools](#tools)
-* [Installation](#installation)
-  * [InternalError when calling webhook: context deadline exceeded](#internalerror-when-calling-webhook-context-deadline-exceeded)
-  * [Invalid header field value](#invalid-header-field-value)
-  * [Helm chart install failure: certificate signed by unknown authority](#helm-chart-install-failure-certificate-signed-by-unknown-authority)
-* [Operations](#operations)
-  * [Stuck runner kind or backing pod](#stuck-runner-kind-or-backing-pod)
-  * [Delay in jobs being allocated to runners](#delay-in-jobs-being-allocated-to-runners)
-  * [Runner coming up before network available](#runner-coming-up-before-network-available)
-  * [Outgoing network action hangs indefinitely](#outgoing-network-action-hangs-indefinitely)
-  * [Unable to scale to zero with TotalNumberOfQueuedAndInProgressWorkflowRuns](#unable-to-scale-to-zero-with-totalnumberofqueuedandinprogressworkflowruns)
-  * [Slow / failure to boot dind sidecar (default runner)](#slow--failure-to-boot-dind-sidecar-default-runner)
+- [Troubleshooting](#troubleshooting)
+  - [Tools](#tools)
+  - [Installation](#installation)
+    - [InternalError when calling webhook: context deadline exceeded](#internalerror-when-calling-webhook-context-deadline-exceeded)
+    - [Invalid header field value](#invalid-header-field-value)
+    - [Helm chart install failure: certificate signed by unknown authority](#helm-chart-install-failure-certificate-signed-by-unknown-authority)
+  - [Operations](#operations)
+    - [Stuck runner kind or backing pod](#stuck-runner-kind-or-backing-pod)
+    - [Delay in jobs being allocated to runners](#delay-in-jobs-being-allocated-to-runners)
+    - [Runner coming up before network available](#runner-coming-up-before-network-available)
+    - [Outgoing network action hangs indefinitely](#outgoing-network-action-hangs-indefinitely)
+    - [Unable to scale to zero with TotalNumberOfQueuedAndInProgressWorkflowRuns](#unable-to-scale-to-zero-with-totalnumberofqueuedandinprogressworkflowruns)
+    - [Slow / failure to boot dind sidecar (default runner)](#slow--failure-to-boot-dind-sidecar-default-runner)
 
 ## Tools
 
 A list of tools which are helpful for troubleshooting
 
-* https://github.com/rewanthtammana/kubectl-fields Kubernetes resources hierarchy parsing tool
-* https://github.com/stern/stern Multi pod and container log tailing for Kubernetes
+- https://github.com/rewanthtammana/kubectl-fields Kubernetes resources hierarchy parsing tool
+- https://github.com/stern/stern Multi pod and container log tailing for Kubernetes
```

Comment:
What's the purpose of chaging the `-` to `*`?

Reply:
in the end no difference both are valid let me put back